content/resources/developer/microservice.md

```diff
@@ -0,0 +1,69 @@
+| fence-service                | /user/                           | https://github.com/uc-cdis/fence                                                           |
+| fenceshib-service            | /                                | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/fenceshib            |
+| google-sa-validation-service | /google-sa-validation-status/    | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/google-sa-validation |
+| grafana                      | /grafana/                        |                                                                                            |
+| guppy-service                | /guppy/                          | https://github.com/uc-cdis/guppy                                                           |
+| hatchery-service             | /lw-workspace/                   | https://github.com/uc-cdis/hatchery                                                        |
+| indexd-service               | /ga4gh/                          | https://github.com/uc-cdis/indexd                                                          |
+| indexd-service               | /index/                          | https://github.com/uc-cdis/indexd                                                          |
+| jupyterhub-service           | /lw-workspace/                   | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/jupyterhub           |
+| jupyterhub-service           | /lw-workspace/hub/logout         | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/jupyterhub           |
+| manifestservice-service      | /manifests/                      | https://github.com/uc-cdis/manifestservice                                                 |
+| peregrine-service            | /peregrine/_status               | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /peregrine/_version              | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/search                      | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/v0/submission/graphql       | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/v0/submission/getschema     | https://github.com/uc-cdis/peregrine                                                       |
+| pidgin-service               | /coremetadata/                   | https://github.com/uc-cdis/pidgin                                                          |
+| portal-service               | /                                | https://github.com/uc-cdis/data-portal                                                     |
```

Comment:
This one is a bit confusing, as we don't have a "portal service"

Reply:
Just pulled these rows from the Nginx config directory, but I see how that could be confusing. Removed portal-service row.